api/organisations/task_helpers.py

```diff
@@ -0,0 +1,137 @@
+import logging
+from datetime import timedelta
+
+from app_analytics.influxdb_wrapper import get_current_api_usage
+from dateutil.relativedelta import relativedelta
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.utils import timezone
+
+from organisations.models import (
+    Organisation,
+    OrganisationAPIUsageNotification,
+    OrganisationRole,
+)
+from users.models import FFAdminUser
+
+from .constants import API_USAGE_ALERT_THRESHOLDS
+
+logger = logging.getLogger(__name__)
+
+
+def send_api_flags_blocked_notification(organisation: Organisation) -> None:
+    recipient_list = FFAdminUser.objects.filter(
+        userorganisation__organisation=organisation,
+    )
+
+    context = {"organisation": organisation}
+    message = "organisations/api_flags_blocked_notification.txt"
+    html_message = "organisations/api_flags_blocked_notification.html"
+
+    send_mail(
+        subject="Flagsmith API use has been blocked due to overuse",
+        message=render_to_string(message, context),
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        recipient_list=list(recipient_list.values_list("email", flat=True)),
+        html_message=render_to_string(html_message, context),
+        fail_silently=True,
+    )
+
+
+def send_api_usage_notification(
```

Comment:
This probably _can_ be a private function now?

Reply:
Sure, I've renamed it.